src/index.ts

```diff
@@ -8,15 +8,30 @@ import {
 import * as readPkgUp from "read-pkg-up";
 import options from "./options";
 const { pkg } = readPkgUp.sync({ cwd: __dirname });
```

Comment:
Change this line to the below
```suggestion
const pkg = readPkgUp.sync({ cwd: __dirname })!.package;
```

Reply:
What is `!.package`?  My editor is telling me the reference `package` is not defined in the current file.  What conflicts in package-lock.json do you see?

Reply:
Disregard that previous comment.  I needed to update and fix some things.  All should be fine now.

Reply:
Yeah sorry, the update to read-pkg-up and removing @types/read-pkg-up prompted this change.